mlrun/api/launcher.py

```diff
@@ -207,6 +207,7 @@ def _save_notifications(self, runobj):
             runobj.spec.notifications,
             runobj.metadata.uid,
             runobj.metadata.project,
+            auth_info=self._auth_info,
```

Comment:
This is confusing but it's only populated in feature store ingestion job.
https://github.com/mlrun/mlrun/blob/development/mlrun/feature_store/ingestion.py#L286
We don't populate this in regular submit_job flow because the `apply_enrichment_and_validation_on_function` is already used before the launch and we don't want to trigger it twice